src/runtime_src/xrt/scheduler/sws.cpp

```diff
@@ -856,7 +858,9 @@ start()
 {
   if (s_running)
     throw std::runtime_error("software command scheduler is already started");
-
+#ifdef EM_DEBUG_SWS

```

Comment:
Please remove or use the XRT_DEBUG infra structure. Let's not pollute code with ifdefs.

Reply:
used XRT_DEBUG infrastructure